### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by skwjdwns** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#207]
Hi.

First of all, thank you for your excellent work.
I would like to ask you a question about loss function. 
In loss.py, loss is calculated by loss = (input - target) * mask.
But doesn't mask have a value of 1 for the background and 0 for the person?
Could you explain more about this part?

Thank you.

**Comment by Daniil-Osokin** [comment]
Hi! This mask makes zero loss for regions with `crowd` annotations: where there are persons, but keypoints were not labeled.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.
